DEVELOPMENT.md

```diff
@@ -0,0 +1,94 @@
+# KCP-Edge Contributor Instructions
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl (version range expected: 1.23-1.25)
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+After cloning this repo (see [instructions](#1-clone-the-kcp-edge-repo)), run the following script to install the required package (Linux or MacOS):
+
+```bash
+hack/install_req.sh
+```
+
+On MacOS you could use the following command as an alternative to the above:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/wls_ubuntu_platform.md)
+
+
+## Setup Instructions
+
+### 1. Clone the kcp-edge repo:
+
+```shell
+git clone https://github.com/kcp-dev/edge-mc.git
+```
+
+
+### 2. Build the kcp-edge binaries:
+
+```shell
+cd edge-mc
+make build
+```
+
+The kcp-edge binaries is created inside the `./bin` directory. Add this directory to your $PATH:
+
+```shell
+export PATH="$PATH:$(pwd)/bin"
+```
+
+### 3. Install KCP
+
+  a) Download kcp binaries for your platform:
+
+  ```shell
+  hack/install-kcp-with-plugins.sh --version v0.11.0 --folder $(pwd)/kcp --create-folder
+  ```
```

Comment:
I would point out that this script is just an automation of the instructions at https://docs.kcp.io/kcp/v0.11/#download-kcp so that the reader knows that KubeStellar does not require anything special and gets good description of what the script does without having to fetch and read the script.